aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+
+======================  =========================    
+Old prefix              New prefix                           
+======================  =========================  
+ASPNET_WEBROOT          ASPNETCORE_WEBROOT
+ASPNET_SERVER           ASPNETCORE_SERVER
+ASPNET_APP              ASPNETCORE_APP
+ASPNET_ENVIRONMENT      ASPNETCORE_ENVIRONMENT
+ASPNET_DETAILEDERRORS   ASPNETCORE_DETAILEDERRORS
+======================  =========================  
+
+The hosting configuration keys are now consistent with the command line, environment variables, and ``hosting.json`` values. The ``Microsoft.AspNet.Hosting.json`` configuration file was renamed to ``hosting.json``.
+
+Namespace and package name changes
+---------------------------------- 
+
+All Microsoft.AspNet.* namespaces are renamed to Microsoft.AspNetCore.*. 
+The EntityFramework.* packages and namespaces are changing to Microsoft.EntityFrameworkCore.*.
+The version numbers of the above are being reset to 1.0.0-*.
+
+Working with IIS
+----------------
+
+Middleware is now setup using ``WebHostBuilder`` and no longer called in the ``Configure`` method of the ``Startup`` class.
+
+Breaking changes in the ``publish-iis`` tool: 
+
+The name of the package that contains the publish-iis tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your project.json file to inlude the "Microsoft.AspNetCore.Server.IISIntegration.Tools" package instead of the "dotnet-publish-iis" package.
+
+The tool needs now to distinguish portable apps from standalone apps to be able to write the ``web.config`` file correctly depending on the application type that is being published. This required adding a new, mandatory parameter ``--framework`` that tells the tool what framework the application was published for.
+
+since HttpPlatformModule was replaced with AspNetCoreModule
+the web.config created by the publish-iis tool now configures IIS to use AspNetCoreModule instead of HttpPlatformHandler to forward requests to Kestrel
+
+The code snippet below shows how to configure the new publish-iis tool in project.json file
+
+.. code-block:: JavaScript
+
+"tools": {
+  "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+    "version": "1.0.0-*",
+    "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
+  }
+},
+"scripts": {
+  "postpublish": "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%"
+}
+
+Additionally, you must turn on server garbage collection in ``project.json`` or ``app.config`` when on full .NET framework.
+
+Changes in MVC
+--------------
+
+- Set option in ``project.json`` to preserve the compilation context if you are doing views.
+- You no longer need to reference the Tag Helper package, it's now referenced by ``Razor`` by default.
+
+There are changes that simplify controller discovery:
+There is a new ``Controller`` attribute that can be used to mark a class and their descendants as controllers.
```

Comment:
> There is ... can be used

Nit: Suggest avoid being verbs (is, will, be, has been) and go for action verbs. This sentence has two.

A new ``Controller`` attribute provides the ability to mark a class and its descendants as controllers.